0.13/build.sbt

```diff
@@ -27,7 +31,7 @@ def commonSettings =  Seq(
           <url>http://typosafe.pl</url>
         </developer>
       </developers>)
-)
+) ++ dependOnSbt("0.13.15")
```

Comment:
yes it does not work. The only problem is that running scripted locally breaks my Ubuntu (sound, internet etc.) Need to play with other options